test/Tester/StreamingTests/EHImplicitSubscriptionStreamRecoveryTests.cs

```diff
@@ -88,14 +46,14 @@ public EHImplicitSubscriptionStreamRecoveryTests()
         public async Task Recoverable100EventStreamsWithTransientErrorsTest()
         {
             logger.Info("************************ EHRecoverable100EventStreamsWithTransientErrorsTest *********************************");
-            await runner.Recoverable100EventStreamsWithTransientErrors(GenerateEvents, ImplicitSubscription_TransientError_RecoverableStream_CollectorGrain.StreamNamespace, 4, 100);
+            await CheckpointSettings.AwaitAndCleanup(runner.Recoverable100EventStreamsWithTransientErrors(GenerateEvents, ImplicitSubscription_TransientError_RecoverableStream_CollectorGrain.StreamNamespace, 4, 100));
```

Comment:
implement IDisposable on EHImplicitSubscriptionStreamRecoveryTests and call cleanup logic from it instead